packages/medusa/src/migrations/1662976816913-order-editing.ts

```diff
@@ -0,0 +1,63 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class orderEditing1662976816913 implements MigrationInterface {
+  name = "orderEditing1662976816913"
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `CREATE TYPE "order_item_change_type_enum" AS ENUM('item_add', 'item_remove', 'quantity_change')`
```

Comment:
it could be but it depends if we will want to define an enum value for each type of change we have to have granularity or we want to have one big "update" change type

Reply:
Do you mean that each changes would create a new item change? Or do you mean an array of values? I though that all changes on one line item would be hold by one item change, since it can exists only one item change per line item in an order edit. And since with have both line we can compute each things that has changed.

Reply:
ok, makes sense, changed `quantity_change` -> `item_update`